python/mscclpp/error_py.cpp

```diff
@@ -31,10 +31,10 @@ void register_error(nb::module_& m) {
       .def(nb::init<const std::string&, int>(), nb::arg("message"), nb::arg("errorCode"));
 
   nb::class_<CudaError, BaseError>(m, "CudaError")
-      .def(nb::init<const std::string&, cudaError_t>(), nb::arg("message"), nb::arg("errorCode"));
+      .def(nb::init<const std::string&, int>(), nb::arg("message"), nb::arg("errorCode"));
```

Comment:
cudaError_t to int?

Reply:
I changed this interface to get an `int` input and cast it into `cudaError_t` internally.